src/data/community-events.json

```
@@ -449,6 +449,15 @@
     "startDate": "2022-09-01",
     "endDate": "2022-09-04"
   },
+  {
+    "title": "ETHEREUMZurich",
```

This seems to be the title most commonly used... can you confirm which is correct?

```suggestion
    "title": "ETHEREUM zurich",
```